holoviews/streams.py

```diff
@@ -595,9 +595,9 @@ def __init__(self, parameterized, parameters=None, watch=True, **params):
         if parameters is None:
             parameters = [p for p in parameterized.params() if p != 'name']
         super(Params, self).__init__(parameterized=parameterized, parameters=parameters, **params)
+        self._memoize = True
         if watch:
```

Comment:
Little bit confused as the the `watch` argument: it doesn't seem to be state (either an attribute or a parameter) but is set through the constructor. Are there cases where you expect users to set `watch=False` even if watch could be supported by the param version?

If not, I would remove this argument so that the param version check is sufficient to know whether watch is being used or not (something you need to determine in `Dynamic` later...)

Reply:
>Are there cases where you expect users to set watch=False even if watch could be supported by the param version?

It's used by the backward compatible `ParamValues` stream.

Reply:
Alternatively, `watch` *could* be state which you would need to check in `Dynamic`...

Reply:
> It's used by the backward compatible `ParamValues` stream.

(GitHub is laggy) In that case, shouldn't `watch` be an attribute to check in `Dynamic`?

Reply:
Don't follow how does this relate to `Dynamic`? That's unrelated.

Reply:
Nevermind, I get it now.